tests/unit/test_recursion_limit.py

```diff
@@ -45,8 +45,8 @@ def test_recursion_to_deep(large_import_chain):
     With the default recursion limit (1000), the recursion error occurs at about 115 modules, with limit 2000
     (as tested below) at about 240 modules, and with limit 5000 at about 660 modules.
     """
-    if is_py37 and is_win:
-        pytest.xfail("worker is know to crash for Py 3.7, 3.8 on Windows")
+    if is_win:
+        pytest.xfail("worker is know to crash on Windows")
```

Comment:
Since you're touching that string, might as well take the opportunity to change that "know" into "known".